CMakeLists.txt

```diff
@@ -3133,22 +3128,19 @@ if(SDL_SHARED)
     # FIXME: Remove SOVERSION in SDL3
     set_target_properties(SDL3 PROPERTIES
       MACOSX_RPATH 1
-      SOVERSION 0
-      OUTPUT_NAME "SDL3-${LT_RELEASE}")
+      SOVERSION 0)
   elseif(UNIX AND NOT ANDROID)
     set_target_properties(SDL3 PROPERTIES
       VERSION ${LT_VERSION}
```

Comment:
You need to prepend with `SOVERSION`.
Otherwise, for 3.2.1 this will create:
```
libSDL3-3.0.so -> libSDL3-3.0.so.0
libSDL3-3.0.so.0 -> libSDL3-3.0.so.3.2.1
libSDL3-3.0.so.3.2.1
```

Reply:
I don't think so? The `LT_VERSION` already starts with the major ABI version (currently 0) which is the same as the `SOVERSION`.

I haven't done the other half of #5626, which is to change the parts of the `LT_VERSION` that come after the `SOVERSION`.

Reply:
You're right. My apologies :)